File: src/codes/Users_Manual/geometry_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
new ControllerBuilder()
   .AddDevice(new AUTD3(new Vector3d(-AUTD3.DeviceWidth, 0, 0)))
   .AddDevice(new AUTD3(Vector3d.Zero))
~;

File: src/codes/Users_Manual/geometry_1.rs
# use autd3::prelude::*;
# 
# #[allow(unused_variables)]
# #[tokio::main]
# async fn main() -> Result<(), Box<dyn std::error::Error>> {
# let _autd = 
Controller::builder([
        AUTD3::new(Vector3::new(-AUTD3::DEVICE_WIDTH, 0., 0.)),
        AUTD3::new(Vector3::zeros())
    ])
#    .open(autd3::link::Nop::builder()).await?;
# Ok(())
# }
